### ISSUE TITLE
Allow for multiple aliases

### ISSUE THREAD

**Issue by cbetta** [tj/commander.js#531]
I'd love for commander allow to allow for multiple aliases. I've found them to be useful for many reasons varying from default subcommands (`records:list` and `records`, `record:show` and `record`), preventing typos (`record:list` vs `records:list`) and just shortcuts (`records:list` and `rl`).

Currently I'm resorting to duplicating my `command` statements which feels far from effective.

**Comment by cbetta** [comment]
I'd be happy to add this but would love feedback on the syntax. 

For one I don't want to show all aliases.

```js
import commander from 'commander';

commander
  .command('link:create <a> <b>')
  .alias('l', 'lc', 'link')
  .action(...);
```

Then `--help` should look something like:

```
> example --help
   Usage: example [options] [command]

   Commands:

      link:create|l|lc|link <a> <b>                         
```

Which might get a bit excessive. Instead some aliases might be hidden instead:

```js
import commander from 'commander';

commander
  .command('link:create <a> <b>')
  .alias('link')
  .hidden_alias('l', 'lc')
  .action(...);
```

Which would then result in:

```
> example --help
   Usage: example [options] [command]

   Commands:
      link:create|link <a> <b>                         
```

The alternative would be to pass an object with the keys of the aliases and the values of the visibility:

```js
import commander from 'commander';

commander
  .command('link:create <a> <b>')
  .alias({ link: true, lc: false, l: false})
  .action(...);
```

**Comment by kodie** [comment]
Would love to see support for multiple aliases.

**Comment by FireCubez** [comment]
Agreed with @tdreid and @kodie , it's the most natural in my opinion

**Comment by ganobrega** [comment]
This is already being supported?

**Comment by shadowspawn** [comment]
@ganobrega Current status is:
- commands support a single alias
- options do not support an alias as such

**Comment by DuckyCrayfish** [comment]
Was a consensus ever reached on the option to support? If not I have yet another suggestion that is a bit of  a middle-ground.

**Comment by shadowspawn** [comment]
Not consensus, throw in another idea @DuckyCrayfish

I personally lean towards just repeating `.alias` or allowing array, and not offering ability to hide. Keep it simple.

For the help, an approach I have copied is to list the aliases in a separate help section. This avoids cluttering up the main command help with the variations. e.g.

```
Aliases:
  i: install
  list: ls
  lsr, list-remote: ls-remote
  remove: rm
```

(The "real" commands are listed on the right.)

**Comment by nrgapple** [comment]
Any update on this?

**Comment by shadowspawn** [comment]
My current thinking is (if we do this at all) allow repeat `.alias` (as suggested) and only the first one appears in built-in help.

So for @kodie example: https://github.com/tj/commander.js/issues/531#issuecomment-382184517

```
commander
  .command('list')
  .alias('ls')
  .alias('l')
  .action(...);
```

```
Usage: index [options] [command]

Options:
  -h, --help      display help for command

Commands:
  list|ls
  help [command]  display help for command
```

**Comment by shadowspawn** [comment]
I have written a PR with support for multiple aliases, the first of which is included in the autogenerated help.

Is this an adequate implementation for the feature request to allow multiple aliases?

**Comment by shadowspawn** [comment]
Support for multiple aliases has been released in Command v5.1.

Thank you for your contributions.
